products/batch_exports/backend/temporal/destinations/postgres_batch_export.py

```diff
@@ -930,6 +939,290 @@ async def insert_into_postgres_activity(inputs: PostgresInsertInputs) -> BatchEx
                 return BatchExportResult(records_completed=details.records_completed)
 
 
+class PostgreSQLConsumerFromStage(ConsumerFromStage):
+    """Consumer for PostgreSQL batch exports using internal stage."""
+
+    def __init__(
+        self,
+        client: PostgreSQLClient,
+        schema: str,
+        table_name: str,
+        schema_columns: list[str],
+    ):
+        super().__init__()
+
+        self.client = client
+        self.schema = schema
+        self.table_name = table_name
+        self.schema_columns = schema_columns
+
+        self.logger.bind(table=f"{schema}.{table_name}")
```

Comment:
`bind` returns a new logger that must be assigned. Also, I'd personally keep schema and table_name separate:
```suggestion
        self.logger = self.logger.bind(table_name=table_name, schema=schema)
```

Reply:
thanks for spotting that. I've fixed it and also fixed it for most of the other destinations, which were also doing it incorrectly.